xmtp_mls/src/await_helper.rs

```diff
@@ -0,0 +1,9 @@
+use futures::Future;
+use tokio::{runtime::Handle, task};
+
+pub fn await_helper<F: Future + Send>(future: F) -> F::Output
+where
+    <F as Future>::Output: Send + 'static,
+{
+    task::block_in_place(move || Handle::current().block_on(future))
+}
```

Comment:
It'd nice if we can start putting helpers/utils into a central place e.g. `xmtp_mls/src/utils/...`

iirc you mentioned runtime issues this may have, wondering if this is for temp usage or considered valid.

Reply:
Hopefully it's for temp usage. @insipx is going to make a `transaction_async` method for the store that will make this obsolete